package.json

```diff
@@ -4,25 +4,37 @@
     "description": "This repository is made by students of Code Chrysalis",
     "main": "index.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "start": "node server.js",
+        "webpack": "webpack --watch",
+        "hack": "webpack-dev-server",
+        "postinstall": "webpack"
     },
     "author": "",
     "license": "MIT",
     "dependencies": {
+        "babel-core": "^6.26.0",
+        "babel-loader": "^7.1.2",
+        "babel-preset-es2015": "^6.24.1",
```

Comment:
save yourself some headache and use `babel-preset-env` the es2015 preset is missing a lot of potential features you might want that were added in es2016, es2017 or later.